database/tables/tables_sqlServer.sql

```diff
@@ -347,42 +347,203 @@ ALTER TABLE [dbo].[QRTZ_TRIGGERS] ADD
+	BEGIN
+		CREATE INDEX [IDX_QRTZ_T_NFT_ST_MISFIRE_GRP]
+		ON [dbo].[QRTZ_TRIGGERS](SCHED_NAME, MISFIRE_INSTR, NEXT_FIRE_TIME, TRIGGER_GROUP, TRIGGER_STATE)
+		WITH (DROP_EXISTING = ON)
+	END
+ELSE
+	BEGIN
+		CREATE INDEX [IDX_QRTZ_T_NFT_ST_MISFIRE_GRP]
+		ON [dbo].[QRTZ_TRIGGERS](SCHED_NAME, MISFIRE_INSTR, NEXT_FIRE_TIME, TRIGGER_GROUP, TRIGGER_STATE)
+	END
+GO
+
+IF EXISTS (SELECT * FROM sys.indexes AS si WHERE si.[name] = 'IDX_QRTZ_FT_INST_JOB_REQ_RCVRY')
+	BEGIN
+		CREATE INDEX [IDX_QRTZ_FT_INST_JOB_REQ_RCVRY]
+		ON [dbo].[QRTZ_FIRED_TRIGGERS](SCHED_NAME, INSTANCE_NAME, REQUESTS_RECOVERY)
+		WITH (DROP_EXISTING = ON)
+	END
+ELSE
+	BEGIN
+		CREATE INDEX [IDX_QRTZ_FT_INST_JOB_REQ_RCVRY]
+		ON [dbo].[QRTZ_FIRED_TRIGGERS](SCHED_NAME, INSTANCE_NAME, REQUESTS_RECOVERY)
+	END
+GO
+
+IF EXISTS (SELECT * FROM sys.indexes AS si WHERE si.[name] = 'IDX_QRTZ_FT_G_J')
+	BEGIN
+		CREATE INDEX [IDX_QRTZ_FT_G_J]
+		ON [dbo].[QRTZ_FIRED_TRIGGERS](SCHED_NAME, JOB_GROUP, JOB_NAME)
+		WITH (DROP_EXISTING = ON)
+	END
+ELSE
+	BEGIN
+		CREATE INDEX [IDX_QRTZ_FT_G_J]
+		ON [dbo].[QRTZ_FIRED_TRIGGERS](SCHED_NAME, JOB_GROUP, JOB_NAME)
+	END
+GO
+
+IF EXISTS (SELECT * FROM sys.indexes AS si WHERE si.[name] = 'IDX_QRTZ_FT_G_T')
+	BEGIN
+		CREATE INDEX [IDX_QRTZ_FT_G_T]
+		ON [dbo].[QRTZ_FIRED_TRIGGERS](SCHED_NAME, TRIGGER_GROUP, TRIGGER_NAME)
+		WITH (DROP_EXISTING = ON)
+	END
+ELSE
+	BEGIN
+		CREATE INDEX [IDX_QRTZ_FT_G_T]
+		ON [dbo].[QRTZ_FIRED_TRIGGERS](SCHED_NAME, TRIGGER_GROUP, TRIGGER_NAME)
+	END
+GO
+
+--CREATE INDEX [IDX_QRTZ_T_G_J]                 ON [dbo].[QRTZ_TRIGGERS](SCHED_NAME, JOB_GROUP, JOB_NAME);
+--CREATE INDEX [IDX_QRTZ_T_C]                   ON [dbo].[QRTZ_TRIGGERS](SCHED_NAME, CALENDAR_NAME);
+--CREATE INDEX [IDX_QRTZ_T_N_G_STATE]           ON [dbo].[QRTZ_TRIGGERS](SCHED_NAME, TRIGGER_GROUP, TRIGGER_STATE);
+--CREATE INDEX [IDX_QRTZ_T_STATE]               ON [dbo].[QRTZ_TRIGGERS](SCHED_NAME, TRIGGER_STATE);
+--CREATE INDEX [IDX_QRTZ_T_N_STATE]             ON [dbo].[QRTZ_TRIGGERS](SCHED_NAME, TRIGGER_NAME, TRIGGER_GROUP, TRIGGER_STATE);
+---CREATE INDEX [IDX_QRTZ_T_NEXT_FIRE_TIME]      ON [dbo].[QRTZ_TRIGGERS](SCHED_NAME, NEXT_FIRE_TIME);
+--CREATE INDEX [IDX_QRTZ_T_NFT_ST]              ON [dbo].[QRTZ_TRIGGERS](SCHED_NAME, TRIGGER_STATE, NEXT_FIRE_TIME);
+--CREATE INDEX [IDX_QRTZ_T_NFT_ST_MISFIRE]      ON [dbo].[QRTZ_TRIGGERS](SCHED_NAME, MISFIRE_INSTR, NEXT_FIRE_TIME, TRIGGER_STATE);
+--CREATE INDEX [IDX_QRTZ_T_NFT_ST_MISFIRE_GRP]  ON [dbo].[QRTZ_TRIGGERS](SCHED_NAME, MISFIRE_INSTR, NEXT_FIRE_TIME, TRIGGER_GROUP, TRIGGER_STATE);
+--CREATE INDEX [IDX_QRTZ_FT_INST_JOB_REQ_RCVRY] ON [dbo].[QRTZ_FIRED_TRIGGERS](SCHED_NAME, INSTANCE_NAME, REQUESTS_RECOVERY);
+--CREATE INDEX [IDX_QRTZ_FT_G_J]                ON [dbo].[QRTZ_FIRED_TRIGGERS](SCHED_NAME, JOB_GROUP, JOB_NAME);
+--CREATE INDEX [IDX_QRTZ_FT_G_T]                ON [dbo].[QRTZ_FIRED_TRIGGERS](SCHED_NAME, TRIGGER_GROUP, TRIGGER_NAME);
+--GO
```

Comment:
Oops this can be deleted. I've split out the original block of drops, to be "ones we are discarding" and "ones we can rebuild our index" in one of two ways.